product_docs/docs/eprs/7/08_xdb_cli/03_xdb_cli_commands/18_adding_tables_to_publication.mdx

```diff
@@ -45,6 +45,11 @@ The `views` parameter applies only for a snapshot-only publication. It's ignored
 !!! Note
     The schema names, table names, and view names that you supply as values for the tables and views parameters are case sensitive. Unless quoted identifiers were used to build the database objects, you must enter Oracle names using uppercase letters (for example, `EDB.DEPT`) and EDB Postgres Advanced Server names in lowercase letters (for example `edb.dept`). See [Quoted identifiers and default case translation](../../10_appendix/04_miscellaneous_xdb_processing_topics/05_quoted_identifiers/#quoted_identifiers) for more information.
 
+
+!!! Note
+   Column mappings are applied when you create subscriptions. If you add a table which qualifies for column mapping with regular expression to the related publication, then Replication Server applies the column mapping for the newly added table. For more information on column mappings, see [Creating a subscription](/36_creating_subscription_cli).
```

Comment:
Should "with regular expression" be "with a regular expression" or "with regular expressions"?

Also, I'm not sure of the significance "with regular expression" here.  I'm wondering if the that clause was removed if the sentence would still be true.  It is probably worth checking with the developers, but I am suggesting that the sentence could simply written as follows and still be valid:

> If you add a table which qualifies for column mapping to the related publication, then Replication Server applies the column mapping for the newly added table.

If so, I would go with that.

Reply:
@anwar-sahib What are your thoughts?

Reply:
We can add column mapping by specifying exact table qualified column name (tableName.columnName) or using regular expression( .*id=integer). As there is more possibility of column name matching on a newly added table when mapping is applied via regular expression as compared to when mapping is provided through table qualified  column name, we had added the clause "with regular expression". 

The simpler version is also true and we can go with the version provided by Matthew.